jenkins/job-configs/kubernetes-jenkins/clean-ci-node-e2e.yaml

```diff
@@ -0,0 +1,18 @@
+- job:
+    name: 'clean-ci-node-e2e'
+    description: 'Clean up leaked node e2e resources. Test owner: test-infra maintainers.'
+    node: ''
+    triggers:
+        - timed: 'H H/3 * * *'
+    properties:
+        - build-discarder:
+            days-to-keep: 14
+    wrappers:
+        - e2e-credentials-binding
```

Comment:
remove this?

(probably should do the same for the PR version. I didn't notice that before.)